rebasehelper/srpm_build_tools/mock_tool.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you to rebase package to the latest version
+# Copyright (C) 2013-2014 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hracek <phracek@redhat.com>
+#          Tomas Hozza <thozza@redhat.com>
+
+import os
+
+from rebasehelper.logger import logger
+from rebasehelper.build_helper import SourcePackageBuildError
+from rebasehelper.build_helper import RpmbuildTemporaryEnvironment
+from rebasehelper.utils import PathHelper
+from rebasehelper.utils import ProcessHelper
+
+
+class MockSRPMBuildTool(object):
+
+    CMD = "mock"
+    DEFAULT = False
+    logs = []
+
+    @classmethod
+    def get_build_tool_name(cls):
+        return cls.CMD
+
+    @classmethod
+    def is_default(cls):
+        return cls.DEFAULT
+
+    @classmethod
+    def do_build_srpm(cls, spec, sources, workdir, results_dir, srpm_builder_options):
```

Comment:
The `do_` prefix is not necessary, it was used in `BuildToolBase` to differentiate the method from already existing `build_srpm`.